docs/DeveloperGuide.md

```diff
@@ -685,13 +685,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * **Medical Condition**: A diagnosis or health issue assigned to a patient, such as "Diabetes Type 2" or "Hypertension." This helps track and manage a patient's health status.
 
-* **Appointment**: A scheduled meeting between a patient and a medical professional, encompassing a specific date, time period and description.
+* **Appointment**: A scheduled meeting between a patient and a doctor, encompassing a specific date, time period and description.
```

Comment:
Good change to ensure our product directly meet the needs of our exact target user!